src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -27,19 +26,15 @@ public Recipe(long id, String recipeName, String description, ArrayList<String>
     this.popularity = popularity;
   }
 
-  public void setImageUrl(String imageUrl) {
-    this.imageUrl = imageUrl;
+  public void setImageBlobKey(String blobKey) {
```

Comment:
because I can't actually render an image right now so I didn't want to make it mandatory for making a Recipe
do you think I should just make it part of the constructor for when I eventually get it working?

Reply:
That makes sense. I think a TODO or comment for yourself so you don't accidentally forget when you do get it running would be helpful.